ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py

```diff
@@ -435,17 +436,20 @@ def __init__(self, instance, sqlusername, sqlpassword, login_as_user, version):
         self.version = version
 
         # DB Connection Object
+        pwd = base64.b64encode(sqlpassword)
+        self.sqlConnection.append("$x = '{}';".format(pwd))
+        self.sqlConnection.append("$p = [System.Text.Encoding]::ASCII.GetString([Convert]::FromBase64String($x));")
         self.sqlConnection.append("$con = new-object "
                                   "('Microsoft.SqlServer.Management.Common.ServerConnection')"
-                                  "'{}', '{}', '{}';".format(instance, sqlusername, sqlpassword))
+                                  '"{}", "{}", "$p";'.format(instance, sqlusername, sqlpassword))
```

Comment:
You have changed placeholder {} to `"$p"` string. As far as I understand, sqlpassword will not be placed here now, and it is redundant argument in format method.
Or am I missing something?